<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskTwoDialog</class>
 <widget class="QDialog" name="TaskTwoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>angle calculator</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>681</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="aLineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="aLabel">
        <property name="text">
         <string>a</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="bLineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="bLabel">
        <property name="text">
         <string>b</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLineEdit" name="alphaLineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="alphaLabel">
        <property name="text">
         <string>alpha</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="degreeRadioButton">
        <property name="text">
         <string>degree</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radRadioButton">
        <property name="text">
         <string>rad</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLCDNumber" name="cLcdNumber"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="calcPushButton">
        <property name="text">
         <string>Calculate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exitPushButton">
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
